tests/backup/backup_test.go

```diff
@@ -71,7 +70,72 @@ var _ = Describe("{BackupClusterVerification}", func() {
 	})
 })
 
-//This testcase verifies basic backup rule,backup location, cloud setting
+// This is a sample test case to verify User/Group Management and role mapping
+var _ = Describe("{UserGroupManagement}", func() {
+	JustBeforeEach(func() {
+		log.Infof("No pre-setup required for this testcase")
+		StartTorpedoTest("Backup: UserGroupManagement", "Creating users and adding them to groups", nil, 0)
+	})
+	It("User and group role mappings", func() {
+		Step("Create Users", func() {
+			err := backup.AddUser("testuser1", "test", "user1", "testuser1@localhost.com", "Password1")
+			if err != nil {
```

Comment:
use log.FailOnError ,so that we can avoid err !=nil check

Reply:
Done. Makes sense.